### DISCUSSION TITLE
ovirt_vm.initialization_custom_script

### DISCUSSION THREAD

**Discussion by NikSudo** [oVirt/terraform-provider-ovirt#494] [Q&A]
What is expecting in `resource.ovirt_vm.initialization_custom_script` ? Bash-scrip? Cloud-init file? 
If cloud-init, do I can use some like `initialization_custom_script = file("/path/to/cloud-init.tpl") `?

**Comment by ghost** [comment]
I believe, that depends on what you have installed inside your VM. If you have installed Cloud-Init, then a file for that. Your VM image needs to know how to get this file and how to execute it.
